Server/ChatService/routes/chat.py

```
@@ -552,6 +624,62 @@ async def send_fcm_async():
 
+            elif message_type == "call_answer":
+                # Handle call answer via WebSocket (instead of HTTP API)
+                from services.call_service import CallService
+                call_id = data.get("call_id")
+                sdp_answer = data.get("sdp_answer")
+                
+                if not call_id or not sdp_answer:
+                    try:
+                        await websocket.send_json({"type": "error", "message": "call_id and sdp_answer are required"})
+                    except Exception:
+                        pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)